docs/gtag-consent-mode.md

```diff
@@ -0,0 +1,7 @@
+## Googla Analytics (gtag) Consent Mode
+
+Unless you're running the [Google Analytics Integration](https://woo.com/products/woocommerce-google-analytics/) extension for a more sophisticated configuration, Google Listings and Ads will add Google's `gtag` to help you track some customer behavior.
```

Comment:
Should we be using `Google Analytics for WooCommerce` here to reflect the updated branding?